features/storage/nvstore/source/nvstore.cpp

```diff
@@ -171,11 +171,56 @@ uint16_t NVStore::get_max_possible_keys()
 
 void NVStore::set_max_keys(uint16_t num_keys)
 {
+    uint16_t key = 0, _old_max_keys = 0;
+
```

Comment:
Please rename to `old_max_keys`. Variables starting with underscores are reserved to class members.